docs/tutorials/sharing-environments.md

```diff
@@ -124,14 +124,38 @@ $ flox push
 ```
 
 !!! note "Note"
-      Right now, only environment owners can push edits to their environments.
+    Right now, only environment owners can push edits to their environments.
 
 
 [flox_push]: ../reference/command-reference/flox-push.md
 [flox_pull]: ../reference/command-reference/flox-pull.md
 [flox_activate]: ../reference/command-reference/flox-activate.md
 [floxhub_concept]: ../concepts/floxhub.md
 
+
+### Pulling a remote environment without connecting to FloxHub
+
+Sometimes you may want to create a FloxHub remote environment that serves as a template starting point for your project but will grow to have different environment definitions across multiple projects. 
```

Comment:
> Sometimes you may want to create a FloxHub remote environment that serves as a template starting point

Depending on who you address i think it might find a broader audience if you said

"Sometimes you want to use a FloxHub remote environment as a template, a starting point for your project."